Project.toml

```
@@ -14,7 +14,7 @@ OffsetArrays = "6fe1bfb0-de20-5000-8ca7-80f57d26f881"
 [compat]
 AbstractFFTs = "1.0"
 FFTW = "1.4"
-ImageFiltering = "0.6"
+ImageFiltering = "0.7"
```

```suggestion
ImageFiltering = "0.6, 0.7"
```
let's see if this fixes the test failures.